src/api/test-controller/assertion.js

```diff
@@ -15,6 +15,7 @@ export default class Assertion {
             message = void 0;
         }
 
+        var allowUnawaitedPromise = options.allowUnawaitedPromise;
```

Comment:
I think, a blank line after this declaration is required to separate it from multi-line declaration with a ternary operator.